docs/README.md

```diff
@@ -23,3 +23,6 @@ Use the Luigi Client API documentation to learn more about the functions and par
 * [Lifecycle](luigi-client-api.md#lifecycle) gives you details on the lifecycle of listeners, navigation nodes, and Event data.
 * [Link Manager](luigi-client-api.md#linkmanager) allows you to navigate to a specific route. 
 * [UX Manager](luigi-client-api.md#uxmanager) helps you to manage the appearance features in Luigi, such as the behavior of backdrop or loading indictors.
+
+## Common
+* [Communication](communication.md) describes how to configure communication-related features.
```

Comment:
```suggestion
* [Communication](communication.md) describes how to configure features related to communication between Luigi Core and Client.
```
I added the bit about Luigi Core and Client, but I also want to ask if the communication is related just to these two communicating with each other or is it a broader term that will include more communication features not limited to Luigi Core - Client relation?